audition/distance_from_best.py

```diff
@@ -33,11 +33,23 @@ def _create(self):
             train_end_time timestamp,
             metric text,
             parameter text,
-            raw_value float,
-            below_best float,
-            below_best_next_time float
+            dist_from_abs_worst float,
```

Comment:
Oh, I actually meant that in some cases that the worst value could literally be positive or negative infinity (think about things like mean-squared error, likelihood, or loss functions), so this distance wouldn't be well-defined in the mathematical sense...

And, yeah, I've been trying to think of how to deal with combining metrics with different directions (or, really, even just with very different scales/magnitudes) in that sort of selection rule and seems like the options are to either do some sort of scaling/normalization or restrict the rule to only allow metrics that go the same direction. The latter is easiest as a stop-gap, of course, but we might want to find a more elegant solution generally. Either way, it feels like that logic should live in the selection rules themselves.